src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -226,16 +228,48 @@ public override async Task<dynamic> GetHostingEnvironmentPostBody(string hosting
             return hostingEnvironmentPostBody;
         }
 
+        public override async Task<DataTable> ExecuteSqlQueryAsync(string cloudServiceName, string query)
+        {
+            if (!query.StartsWith("\""))
+            {
+                // BUG: Passing as JSON requires query to be wrapped in quotes, fix API expected content type
+                query = $"\"{query}\"";
+            }
+
+            var request = new HttpRequestMessage(HttpMethod.Post, $"/api/service/{cloudServiceName}/invokesql?api-version=2.0")
+            {
+                Content = new StringContent(query, Encoding.Default, "application/json")
+            };
+
+            var response = await SendObserverRequestAsync(request);
```

Comment:
I remember sometime ago during our end of scrum demo we had discussions about parameterized queries. Will this PR take care of it or do we have any plans for it.

Reply:
I would like parameterized queries, but this PR doesn't address them at all. We can add parameters as needed, but for now this solves the main issue of too much data coming back from Observer.

Reply:
Okay